07-reputation/02-managers.md

```diff
@@ -0,0 +1,18 @@
+## Managers(mother nodes)
+
+Exactly one manager is selected for each node in each epoch. Every manager is also a child node of another manager. Thus, each network participant plays two roles: a child node for one of the managers, and a manager for one of the child nodes.
+
+Managers are required to assess the trust of a child node considering its reputation among other nodes. Also, managers are direct (and only) participants in the iterative part of the EigenTrust algorithm; in the late stage of the algorithm, they send Global Trust in their child node to the Reputation contract.
+
+### Defining a manager for a node
+
+To unambiguously determine the manager for the current node, the following rule is used:
+
+1. For the current epoch, a network map (a list of active participants) is obtained from the `Netmap` of the contract.
+2. The resulting array is sorted using [HRW](https://github.com/nspcc-dev/hrw) hashing, which takes into account the current epoch number.
```

Comment:
There is a Glossary section for HRW

Reply:
moved `HRW` from `.md` file to `.tex` file. deleted direct http links in text